fr.free.Qccrypt.yml

```diff
@@ -0,0 +1,53 @@
+app-id: fr.free.Qccrypt
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: qccrypt
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
if it supports wayland then it's `fallback-x11` on line 8

Reply:
Thank you for your help. To be honest, I reused a YML file from another flathub project, and it was written like this. So, if I understand, I have to remove the line #8 as my application supports Wayland, that's right?

Reply:
Ok, sorry, I read wrong. So, if I understand, I have to replace the line `` - --socket=x11`` with `` - --socket=fallback-x11``, that's right?